services/galley/src/Galley/API/Create.hs

```diff
@@ -98,17 +99,21 @@ internalCreateManagedConversation zusr zcon (NewConvManaged body) = do
 createRegularGroupConv :: UserId -> ConnId -> NewConvUnmanaged -> Galley ConversationResponse
 createRegularGroupConv zusr zcon (NewConvUnmanaged body) = do
   name <- rangeCheckedMaybe (newConvName body)
-  _uids <- checkedConvSize (newConvUsers body) -- currently not needed, as we only consider local IDs
+  localDomain <- viewFederationDomain
   let localUserIds = newConvUsers body
+      qualifiedUserIds = newConvQualifiedUsers body
   ensureConnected zusr localUserIds
-  localCheckedUsers <- checkedConvSize localUserIds
+  let allUsers = map (`Qualified` localDomain) localUserIds <> qualifiedUserIds
+  checkedUsers <- checkedConvSize allUsers
```

Comment:
The connection check `ensureConnected` is lost here when doing the check only on non-qualified users. That check also needs to happen on all qualified users with the localDomain. (A test to that effect would be great, too). A FUTUREWORK should be added to also do that connection check once we have federated connections.

Reply:
Just to confirm: @jschaul , this is resolved now, right?

Reply:
Yes, that is resolved now (for local users). Remote users and connections to remote users will come later.